examples/smb-fMRI.ipynb

```diff
@@ -0,0 +1,316 @@
+    "        else:\n",
+    "            mask_data = mask_data.astype(bool)\n",
+    "    else:\n",
+    "        mask_data = mask.get_data().astype(bool)\n",
+    "\n",
+    "    #get voxel coordinates\n",
+    "    R = np.float64(np.argwhere(mask_data))\n",
+    "\n",
+    "    #get scanner RAS coordinates based on voxel coordinates\n",
+    "    if affine is not []:\n",
+    "        R = (np.dot(affine[:3,:3], R.T) + affine[:3,3:4]).T\n",
+    "\n",
+    "    #get ROI data, and run preprocessing\n",
+    "    nifti_masker = NiftiMasker(mask_img=mask, standardize=zscore, detrend=detrend, smoothing_fwhm=smoothing_fwmw)\n",
+    "    img = nib.load(nifti_file)\n",
+    "    all_images = np.float64(nifti_masker.fit_transform(img))\n",
+    "    data = all_images.T.copy()\n",
+    "\n",
+    "    #save data\n",
+    "    return data, R\n",
+    "\n",
+    "\n",
+    "# get brain data\n",
+    "neurodata1,R1 = extract_data('PATH_TO_DOWNLOADED_DATA.nii.gz')\n",
+    "\n",
+    "# to make TFA quick for testing, only do it on the TRs where we have behavior. To do this, first load behavior: \n",
+    "behavdata = pd.read_table('PATH_TO_DOWNLOADED_DATA.tsv')\n",
+    "\n",
+    "# figure out which neurodata had behavior in it\n",
+    "behavdataTRs = (behavdata.onset/2).values.astype(int)\n",
+    "# subset neuro data\n",
+    "neurodata = neurodata[:,behavdataTRs]\n",
+    "\n",
+    "# Run TFA\n",
+    "n_voxel, n_tr = neurodata.shape\n",
+    "tfa = TFA(K=5, max_num_voxel=int(n_voxel*0.5), max_num_tr=int(n_tr*0.5), verbose=True)\n",
+    "tfa.fit(neurodata, R)\n",
+    "# get weights (and transpose to have standard format)\n",
+    "brainWeights = tfa.W_.T\n",
+    "\n",
+    "# extract behavior data\n",
+    "behavdata = np.array([behavdata.response_time.values,(behavdata.correctness=='correct').values.astype(int), behavdata.StimVar.values])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "DDM parameters have finite support. Infinite support makes it much easier to sample and optimize, so we write some transformations here (maybe these should live in wfpt_py?)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# get a vec'd likelihood\n",
+    "wfpt_logp_vec = np.vectorize(wfpt_logp)\n",
+    "\n",
+    "def wfptParamTransform(x0, t0, cong_a, incong_a, cong_z, incong_z):\n",
+    "    # go from infinite support to where we actually want to be\n",
+    "    t0 = np.exp(t0)\n",
+    "    cong_a = np.exp(cong_a)\n",
+    "    incong_a = np.exp(incong_a)\n",
+    "    cong_z = np.exp(cong_z)\n",
+    "    incong_z = np.exp(incong_z)\n",
+    "    return x0, t0, cong_a, incong_a, cong_z, incong_z\n",
+    "\n",
+    "def wfptParamUntransform(x0, t0, cong_a, incong_a, cong_z, incong_z):\n",
+    "    # go from infinite support to where we actually want to be\n",
```

Comment:
Please, correct the comment here. Is the same as above but the function is doing the opposite.